### PR TITLE
Normalized test names and file names

### PR DESCRIPTION
**Pull request by AustinSanders**
Changes the file and test names to better match the guidelines described [here](https://github.com/USGS-Astrogeology/ISIS3/wiki/Writing-ISIS3-Tests-Using-Gtest-and-Ctest#app-conversion--gtest-pr-checklist)

## Related Issue
#4206 

## Motivation and Context
Test names should be consistent

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [x] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/tests/FunctionalTestsCamdev.cpp [hunk 1]
```diff
@@ -13,7 +13,7 @@ using namespace Isis;
 
 static QString APP_XML = FileName("$ISISROOT/bin/xml/camdev.xml").expanded();
 
-TEST_F(LineScannerCube, FunctionalTestsCamDevDefault) {
+TEST_F(LineScannerCube, FunctionalTestCamDevDefault) {
```
Comment (scsides) [inline isis/tests/FunctionalTestsCamdev.cpp]: If I'm following the normalization, shouldn't this be "FunctionalTestCamdevDefault" ?

File: isis/tests/FunctionalTestsMoccal.cpp [hunk 2]
```diff
@@ -12,10 +12,10 @@ using namespace Isis;
 
 static QString APP_XML = FileName("$ISISROOT/bin/xml/moccal.xml").expanded();
 
-TEST_F(MgsMocCube, FunctionalTestMroMoccalDefault) {
+TEST_F(MgsMocCube, FunctionalTestMromoccalDefault) {
```
Comment (scsides) [inline isis/tests/FunctionalTestsMoccal.cpp]: Not sure how this got through the first time. "moccal" is an MGS app not an MRO. On the other hand, the app name is "moccal" not "mgsmoccal", shouldn't this just be "FunctionalTestMoccalDefault" ?

File: isis/tests/FunctionalTestsMoccal.cpp [hunk 3]
```diff
@@ -40,18 +40,18 @@ TEST_F(MgsMocCube, FunctionalTestMroMoccalDefault) {
...
+
   EXPECT_DOUBLE_EQ(oCubeStats->Average(), 0.056909484090283513);
   EXPECT_DOUBLE_EQ(oCubeStats->Sum(), 22.763793636113405);
   EXPECT_DOUBLE_EQ(oCubeStats->ValidPixels(), 400);
   EXPECT_DOUBLE_EQ(oCubeStats->StandardDeviation(), 0.0021719888294085255);
 }
 
 
-TEST_F(MgsMocCube, FunctionalTestMroMoccalIofFalse) {
+TEST_F(MgsMocCube, FunctionalTestMromoccalIofFalse) {
```
Comment (scsides) [inline isis/tests/FunctionalTestsMoccal.cpp]: ditto

File: isis/tests/FunctionalTestsMoccal.cpp [hunk 4]
```diff
@@ -85,10 +85,10 @@ TEST_F(MgsMocCube, FunctionalTestMroMoccalIofFalse) {
 }
 
 
-TEST_F(MgsMocCube, FunctionalTestMroMoccalNullwagoTrue) {
+TEST_F(MgsMocCube, FunctionalTestMromoccalNullwagoTrue) {
```
Comment (scsides) [inline isis/tests/FunctionalTestsMoccal.cpp]: ditto

File: isis/tests/FunctionalTestsMoccal.cpp [hunk 5]
```diff
@@ -121,7 +121,7 @@ TEST_F(MgsMocCube, FunctionalTestMroMoccalNullwagoTrue) {
 }
 
 
-TEST_F(MgsMocCube, FunctionalTestMroMoccalCameraComparison) {
+TEST_F(MgsMocCube, FunctionalTestMromoccalCameraComparison) {
```
Comment (scsides) [inline isis/tests/FunctionalTestsMoccal.cpp]: ditto

**Review by scsides — CHANGES_REQUESTED** [review]

**Review by Kelvinrr — COMMENTED** [review]
Other than the ones Stuart found, I couldn't find other inconsistencies.

**Comment by Kelvinrr** [comment]
@scsides Does this look good to go?

**Review by scsides — APPROVED** [review]

**Pull request merged**